python/cuxfilter/tests/assets/test_gpu_histogram.py

```diff
@@ -8,7 +8,54 @@
 from cuxfilter.charts.core.core_chart import BaseChart
 
 
-def test_calc_value_counts():
+@pytest.mark.parametrize(
+    "custom_binning, result",
+    [
+        (True, np.array([[0, 1, 2, 3, 7, 8], [100, 150, 300, 100, 50, 150]]),),
+        (
+            False,
+            np.array(
+                [
+                    [
+                        1,
+                        5,
+                        10,
+                        11,
+                        15,
+                        22,
+                        23,
+                        25,
+                        27,
+                        30,
+                        35,
+                        39,
+                        99,
+                        104,
+                        109,
+                    ],
+                    [
+                        50,
+                        50,
+                        50,
+                        50,
+                        50,
+                        50,
+                        100,
+                        50,
+                        50,
+                        50,
+                        50,
+                        50,
+                        50,
+                        50,
+                        100,
+                    ],
+                ]
+            ),
+        ),
+    ],
+)
```

Comment:
nit: sometimes black formatting is alot :) This might collapse a bit if the lists are pulled out in to variables to pass to the decorator

Reply:
fixed